2023-09-07: Handbook: Communications: Shared calendars (#13636)
fleetdm/fleet: handbook/company/communications.md

<blockquote puropse="large-quote">In an all-remote company, "face time" matters. Remember: even if someone's calendar is open, they have other work to do. Limiting (or batching up) internal meetings can enable longer, uninterrupted stretches of deep work.</blockquote>

### Modifying an event organized by someone else
To edit an event where someone else at Fleet is the organizer, you can first subscribe to their calendar in Google Calendar and then edit the event on their calendar.  Your edits will automatically apply to all attendees.
This works because every Fleetie grants edit access to everyone else at Fleet as part of onboarding.

### Shared calendars

> At Fleet, our shared team calendars are more than just schedules; they are the heartbeats of our operations. Acting as the primary source for team rituals, they facilitate the smooth execution of each sprint.

Reviewer: Let's cut the first sentence (less surface area).  Victory is when we've cut down the words on this page like 50%

> Looking to add, change, or remove a shared calendar? [Create an issue](https://github.com/fleetdm/fleet/issues/new) with your idea, title it "Calendar idea", write up your idea, and label it with `#g-ceo` so that it can be brought to the e-group or appropriate DRI for feedback.

Reviewer: ```suggestion
Looking to add, change, or remove a shared calendar? [Create an issue for the CEO](https://fleetdm.com/handbook/ceo#contact-us) and the appropriate DRI will reply with feedback.
```

### Modifying an event organized by someone else
To edit an event where someone else at Fleet is the organizer, you can first subscribe to their calendar in Google Calendar and then edit the event on their calendar.  Your edits will automatically apply to all attendees.
This works because every Fleetie grants edit access to everyone else at Fleet as part of onboarding.

### Shared calendars
At Fleet, our shared team calendars are more than just schedules; they are the heartbeats of our operations. Acting as the primary source for team rituals, they facilitate the smooth execution of each sprint.

Looking to add, change, or remove a shared calendar? Follow this process:
- **Define the goal:** identify and articulate how adjustments will benefit or align with current practices.
- **Engage your manager:** seek their insight and endorsement.
- **Seek final approval:** ask your manager to pitch the idea at the e-group weekly meeting for the CEO's approval.

> - **Implement and inform:** set clear agendas, facilitate gong recordings, remove duplicates, and keep key stakeholders informed.

Reviewer: Hey Isabell two things:

1. What if we replaced all four of these steps with: 
> [Create an issue](https://github.com/fleetdm/fleet/issues/new) with your idea, title it "Calendar idea", write up your idea, and label it with `#g-ceo` so that it can be brought to the e-group or appropriate DRI for feedback.

2. Much more minor, and irrelevant if we make the change I'm suggesting here, but we've been going with the convention of capitalized words after colons in these contexts:
<img width="605" alt="image" src="https://github.com/fleetdm/fleet/assets/618009/138a5c89-8ed2-4707-baf3-83381d30f646">

Author: @mikermcneil I like that idea!
